Index: Sudoku.Recuit/Ressources/SelfCallSolver.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import clr\r\nclr.AddReference(\"Sudoku.Shared\")\r\nclr.AddReference(\"Sudoku.SodukuRecuitSolver\")\r\nfrom Sudoku.RecuitSolver import SudokuRecuitSolver\r\nnetSolver = SudokuRecuitSolver()\r\nsolvedSudoku = netSolver.Solve(sudoku)
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Sudoku.Recuit/Ressources/SelfCallSolver.py b/Sudoku.Recuit/Ressources/SelfCallSolver.py
--- a/Sudoku.Recuit/Ressources/SelfCallSolver.py	
+++ b/Sudoku.Recuit/Ressources/SelfCallSolver.py	
@@ -1,6 +1,6 @@
 import clr
 clr.AddReference("Sudoku.Shared")
-clr.AddReference("Sudoku.SodukuRecuitSolver")
-from Sudoku.RecuitSolver import SudokuRecuitSolver
+clr.AddReference("Sudoku.RecuitSolver")
+from Sudoku.Recuit import SudokuRecuitSolver
 netSolver = SudokuRecuitSolver()
 solvedSudoku = netSolver.Solve(sudoku)
\ No newline at end of file
